.gitignore

```
@@ -25,4 +25,5 @@ vite.config.ts.timestamp-*
 
 # Generation
 generation/.cache
-__pycache__
\ No newline at end of file
+*.cpython-312.pyc
```

```suggestion
*.py[cod]
```

See https://github.com/github/gitignore/blob/main/Python.gitignore